locales/en/public.json

```diff
@@ -55,6 +55,11 @@
       "LANGUAGE_REGION": "Language & Region",
       "NOTIFICATION_MESSAGE": "Notifications & Messages"
     },
+    "CHARTS_CONFIG": {
+      "DESCRIPTION": "Set configuration options for the behaviour of Metrics cards on the Dashboard.",
```

Comment:
Wondering if we should set this to empty? Right now, it looks a little odd. I guess the title is already self-explained and description is already available for each config group?

![Screenshot from 2023-02-09 01-20-36](https://user-images.githubusercontent.com/68053619/217733859-523a7eb5-4a7b-4c10-aff9-a5be8965a056.png)